core/src/avm2/globals/flash/events/AVHTTPStatusEvent.as

```diff
@@ -0,0 +1,40 @@
+package flash.events
+{
+    
+    public class AVHTTPStatusEvent extends Event
+    {
+        public static const AV_HTTP_RESPONSE_STATUS:String = "avHttpResponseStatus"; // Unlike the httpStatus event, the httpResponseStatus event is delivered before any response data.
+
+        private var _status: int; // The HTTP status code returned by the server.
+        public var responseURL: String; // The URL that the response was returned from.
+        public var responseHeaders: Array; // The response headers that the response returned, as an array of URLRequestHeader objects.
+
+        public function AVHTTPStatusEvent(type:String, bubbles:Boolean = false, cancelable:Boolean = false, status:int = 0, responseUrl:String = null, responseHeaders:Array = null)
+        {
+            super(type,bubbles,cancelable);
+            this._status = status;
+            this.responseURL = responseUrl;
+            this.responseHeaders = responseHeaders;
+        }
+        
+
+        //  Creates a copy of the AVHTTPStatusEvent object and sets the value of each property to match that of the original.
+        override public function clone():Event
+        {
+            return new AVHTTPStatusEvent(this.type, this.bubbles, this.cancelable, this.status, this.responseURL, this.responseHeaders);
+        }
+
+        //  Returns a string that contains all the properties of the AVHTTPStatusEvent object.
+        override public function toString():String
+        {
+            return this.formatToString("AVHTTPStatusEvent","type","bubbles","cancelable","eventPhase","status");
```

Comment:
Have you verified the order of these properties in FP?

Reply:
No, I just looked at the "format" piece of code from the documentation, the reason I added the "eventPhase" to these was because I noticed it added in already existing event code. I now have a proper AS3 and FP Dev Environment set up to check these against what FP spits out and for this event class in particular it does spits out the `responseURL` and `responseHeaders`.

Reply:
Ah, didn't notice that line in the docs for `toString`. That's good as a starting point. And again, I think it's fine to assume that it is correct (although, you're saying that in this example it is in fact not?), and then fix these minor things later when they come up.

Reply:
(It's also these things, along with whether `clone()` leaves something out, that could use some testing via comparison with FP, but maybe this too, is a "perfect is the enemy of good" situation, as I read recently on Discord... :) )